pkg/webhook/admission/deployment/model_initializer_injector.go

```diff
@@ -137,6 +137,13 @@ func (mi *ModelInitializerInjector) InjectModelInitializer(deployment *appsv1.De
 	}
 	userContainer.VolumeMounts = append(userContainer.VolumeMounts, sharedVolumeReadMount)
 
+	// Change the CustomSpecModelUri env variable value to the default model path if present
+	for index, envVar := range userContainer.Env {
+		if envVar.Name == constants.CustomSpecModelUriEnvVarKey && envVar.Value != "" {
+			userContainer.Env[index].Value = constants.DefaultModelLocalMountPath
+		}
+	}
```

Comment:
I have no idea why L140-L145 is in need. It seems they can be dropped. Please correct me if you don't agree with me

Reply:
The user relies on this env variable in his custom script to load the models from the mount location.

Reply:
OK, SGTM! thanks!

Reply:
Magic environment variables don't SGTM ;)

Reply:
Good to see this optional